### PR TITLE
Adding option to vic-machine to specify default bridge network range

### PR DESCRIPTION
**Pull request by hmahmood**

### CODE REVIEW

**Comment by andrewtchin** [comment]
lgtm

File: lib/portlayer/network/network.go [hunk 1]
```diff
@@ -40,19 +40,27 @@ func (e DuplicateResourceError) Error() string {
 func Init() error {
 	var err error
 
-	bridgeRange := net.IPNet{
-		IP:   net.IPv4(172, 17, 0, 0),
-		Mask: net.CIDRMask(12, 32),
+	bridgeRange := Config.BridgeIPRange
+	if bridgeRange == nil || len(bridgeRange.IP) == 0 || bridgeRange.IP.IsUnspecified() {
+		_, bridgeRange, err = net.ParseCIDR("172.16.0.0/12")
```
Comment (caglar10ur) [inline lib/portlayer/network/network.go]: We go from .17 to .16, guessing that was intentional?

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

File: lib/portlayer/network/network.go (hunk 1 above)
Comment (hmahmood) [inline lib/portlayer/network/network.go]: Actually in this case it doesn't matter since the mask is /12; the network will still be 172.16.0.0.

**Comment by mdubya66** [comment]
lgtm

**Comment by hmahmood** [comment]
♻️

**Pull request merged**